demo/clusters/kind/scripts/create-kind-cluster.sh

```diff
@@ -28,3 +28,7 @@ kind create cluster \
 	--name "${KIND_CLUSTER_NAME}" \
 	--image "${KIND_IMAGE}" \
 	--config "${KIND_CLUSTER_CONFIG_PATH}"
+
+# Unmount the masked /proc/driver/nvidia to allow
+# dynamically generated MIG devices to be discovered
+docker exec -it "${KIND_CLUSTER_NAME}-worker" umount -R /proc/driver/nvidia
```

Comment:
Question: If we move to a situation where this is not mounted, will this error out?

Reply:
Yes, this will cause an error

Reply:
Do we want to fail silently or with a warning instead? (Sorry should have added this in the first question already).